src/Kernel/Magic/WhoMagic.cs

```diff
@@ -23,7 +23,26 @@ public class WhoMagic : AbstractMagic
             "who",
             new Documentation
             {
-                Summary = "Provides actions related to the current workspace."
+                Summary = "Lists the Q# operations available in the current session.",
+                Description = @"
+                    This magic command returns a list of Q# operations and functions that are available
```

Comment:
I think it may also list UDTs that are available for use with interactively defined functions and operations, but I'm not sure.